ocs_ci/ocs/utils.py

```diff
@@ -1012,3 +1013,24 @@ def revive_osd_external(ceph_cluster, osd_id):
         except CommandFailed:
             log.error("Failed to revive osd")
             raise
+
+
+def reboot_node(ceph_node, timeout=300):
+    """
+    Reboot a node with given ceph_node object
+
+    Args:
+        ceph_node (CephNode): ceph node object representing the node
+        timeout (int): Wait time for node to comeback
+
+    """
+    ceph_node.exec_command(
+        cmd='reboot',
+        check_ec=False,
+        long_running=True
```

Comment:
```suggestion
        long_running=True,
```

Black will add it as well I think so maybe you can do it in this PR directly ;)